src/pages/CreateUserPage.jsx

```diff
@@ -455,6 +460,7 @@ function CreateUserPage(props) {
                                                 <option value="PUBLISHER">Publicador</option>
                                             )}
                                             {user.role === 'ADMIN' && <option value="COORDINATOR">Coordenador</option>}
+                                            {user.role === 'ADMIN' && <option value="ADMIN">Admin</option>}
                                         </select>
```

Comment:
```suggestion
                                            {user.role === 'ADMIN' && <option value="COORDINATOR">Coordenador</option>}
                                        </select>
```
- Estou descobrindo agora que a API permite criar admins via endpoints. É uma falha de validação grave que deverá ser corrigida com urgência. Portanto, convém que o front mantenha a limitação;